packages/console/src/pages/Connectors/index.tsx

```diff
@@ -42,30 +67,60 @@ const Connectors = () => {
               <td colSpan={2}>loading</td>
             </tr>
           )}
-          {data?.map(({ id, metadata: { name, logo }, enabled }) => (
-            <tr key={id}>
+          {!isLoading && !isSocial && (
+            <tr>
               <td>
-                <Link to={`/connectors/${id}`} className={styles.link}>
-                  <ItemPreview
-                    title={name[i18n.language] ?? name.en ?? '-'}
-                    subtitle={id}
-                    icon={
-                      logo.startsWith('http') ? (
-                        <img className={styles.logo} src={logo} />
-                      ) : (
-                        <ImagePlaceholder />
-                      )
-                    }
-                  />
-                </Link>
+                <ConnectorName
+                  connector={emailConnector}
+                  titlePlaceholder={t('connectors.type.email')}
+                />
               </td>
+              <td>{t('connectors.type.email')}</td>
               <td>
-                {enabled
-                  ? t('connectors.connector_status_enabled')
-                  : t('connectors.connector_status_disabled')}
+                {emailConnector ? (
+                  <Status status="operational">{t('connectors.connector_status_enabled')}</Status>
+                ) : (
+                  <Button title="admin_console.connectors.set_up" />
+                )}
               </td>
             </tr>
-          ))}
+          )}
+          {!isLoading && !isSocial && (
+            <tr>
+              <td>
+                <ConnectorName
+                  connector={smsConnector}
+                  titlePlaceholder={t('connectors.type.sms')}
+                />
+              </td>
+              <td>{t('connectors.type.sms')}</td>
+              <td>
+                {smsConnector ? (
+                  <Status status="operational">{t('connectors.connector_status_enabled')}</Status>
+                ) : (
+                  <Button title="admin_console.connectors.set_up" />
+                )}
+              </td>
+            </tr>
+          )}
+          {isSocial &&
+            socialConnectors?.map((connector) => (
+              <tr key={connector.id}>
+                <td>
+                  <ConnectorName connector={connector} />
+                </td>
+                <td>{t('connectors.type.social')}</td>
+                <td>
+                  <Status status={connector.enabled ? 'operational' : 'offline'}>
+                    {t(
+                      connector.enabled
+                        ? 'connectors.connector_status_enabled'
+                        : 'connectors.connector_status_disabled'
+                    )}
+                  </Status>
```

Comment:
we can do one more layer dividing for social / non-social connectors tab, say `<EmailAndSmsConnector>` and `<SocialConnectors>`

Reply:
updated, they are now in 2 variables instead of components, will refactor into components later, maybe in LOG-1752

Reply:
cool